docs/index.html

```diff
@@ -1,619 +1 @@
-
-<span class="k">let</span> <span class="nv">doubled</span> <span class="o">=</span> <span class="n">doc</span><span class="o">.</span><span class="n">map</span> <span class="p">{</span> <span class="n">elem</span> <span class="o">-&gt;</span> <span class="kt">Int</span> <span class="k">in</span>
-    <span class="k">guard</span> <span class="k">case</span> <span class="k">let</span> <span class="nv">value</span> <span class="o">=</span> <span class="o">.</span><span class="nf">int64</span><span class="p">(</span><span class="n">value</span><span class="p">)</span> <span class="k">else</span> <span class="p">{</span>
-        <span class="k">return</span> <span class="mi">0</span>
-    <span class="p">}</span>
-
-    <span class="k">return</span> <span class="kt">Int</span><span class="p">(</span><span class="n">value</span> <span class="o">*</span> <span class="mi">2</span><span class="p">)</span>
-<span class="p">}</span>
-<span class="nf">print</span><span class="p">(</span><span class="n">doubled</span><span class="p">)</span> <span class="c1">// prints `[2, 4, 6, 8]`</span>
-</code></pre>
-
-<p>Note that <code><a href="Structs/Document.html">Document</a></code> conforms to <code>Collection</code>, so useful methods from
-<a href="https://developer.apple.com/documentation/swift/sequence"><code>Sequence</code></a> and
-<a href="https://developer.apple.com/documentation/swift/collection"><code>Collection</code></a> are
-all available. However, runtime guarantees are not yet met for many of these
-methods.</p>
-<h3 id='usage-with-kitura-vapor-and-perfect' class='heading'>Usage With Kitura, Vapor, and Perfect</h3>
-
-<p>The <code>Examples/</code> directory contains sample projects that use the driver with <a href="https://github.com/mongodb/mongo-swift-driver/tree/master/Examples/Kitura">Kitura</a>, <a href="https://github.com/mongodb/mongo-swift-driver/tree/master/Examples/Vapor">Vapor</a>, and <a href="https://github.com/mongodb/mongo-swift-driver/tree/master/Examples/Perfect">Perfect</a>.</p>
-<h2 id='development-instructions' class='heading'>Development Instructions</h2>
-
-<p>See our <a href="https://mongodb.github.io/mongo-swift-driver/development.html">development guide</a> for instructions for building and testing the driver.</p>
-
-          </section>
-        </section>
-        <section id="footer">
-          <p>&copy; 2019 <a class="link" href="" target="_blank" rel="external">Matt Broadstone, Kaitlin Mahar, and Patrick Freed</a>. All rights reserved. (Last updated: 2019-12-18)</p>
-          <p>Generated by <a class="link" href="https://github.com/realm/jazzy" target="_blank" rel="external">jazzy ♪♫ v0.13.0</a>, a <a class="link" href="https://realm.io" target="_blank" rel="external">Realm</a> project.</p>
-        </section>
-      </article>
-    </div>
-  </body>
-</div>
-</html>
+<html><head><meta http-equiv="refresh" content="0; url=mongoswift/index.html" /></head></html>
```

Comment:
copied from NIO. this makes it so if you go to the main documentation URL it forwards you to the MongoSwift docs main page (which has the README listing the modules).